images/win/scripts/Installers/Install-Chrome.ps1

```diff
@@ -66,7 +66,7 @@ $ChromeDriverZipDownloadUrl = "https://chromedriver.storage.googleapis.com/${Chr
 $ChromeDriverArchPath = Start-DownloadWithRetry -Url $ChromeDriverZipDownloadUrl -Name $ChromeDriverArchName
 
 Write-Host "Expand Chrome WebDriver archive..."
```

Comment:
Maybe we can get rid of such messages to avoid duplication?

Reply:
Good point. Probably it makes sense to delete `Write-Host "Expand archive '$PATH' to '$DestinationPath' directory"` line